test/tests/tsdx-build-closure-compiler.test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @jest-environment node
+ */
+
+const shell = require('shelljs');
+const util = require('../fixtures/util');
+
+shell.config.silent = false;
+
+const stageName = 'stage-build-closure-compiler';
+
+const cjsBundleDev = 'dist/build-withconfig.cjs.development.js';
+const cjsBundleProd = 'dist/build-withconfig.cjs.production.min.js';
+const esmBundle = 'dist/build-withconfig.esm.js';
+
+const simplePattern = /exports\.signature=signature/;
+const advancedPattern = /exports\.a="bar 0"/;
```

Comment:
huh... doesn't this minification break the usage?? since signature is no longer exported. is that a bug in closure or is there a way to disable exports minification?

Reply:
@agilgur5  hmm.. this is not good.. need to investigate (a lot of ADVANCED options)

Reply:
Ah, I haven't looked the API at all. Keywords would likely be "exports" and "mangle" as I believe the technical term for changing variable names is "mangling"